src/main/java/com/lothrazar/cyclic/block/cable/energy/TileCableEnergy.java

```diff
@@ -64,8 +67,12 @@ private void tryExtract(Direction extractSide) {
     if (itemHandlerFrom == null)
       return;
 
+    final int capacity = energy.getMaxEnergyStored() - energy.getEnergyStored();
```

Comment:
I'm checking the capacity as otherwise, you can set up cables so they keep going from full to no energy when (for instance) filling a battery.